.github/workflows/rtd.yml

```diff
@@ -0,0 +1,21 @@
+name: Update readthedocs branch
+
+on:
+  push:
+    branches: ["branch-0.20"]
```

Comment:
Like before, noting that this needs to be changed with every release, which can be done with a release script like this:

- https://github.com/rapidsai/cudf/blob/branch-0.20/ci/release/update-version.sh#L44-L75

Reply:
Sounds good, I don't have a preference, but for now we can do this manually and later have a script. Either way is fine by me.